app/build.gradle

```diff
@@ -30,13 +36,11 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-//        coreLibraryDesugaringEnabled true
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
```

Comment:
Why downgrade? 

We've had CI issues when running older Java versions.

Reply:
It's code from app-old. I have 17 version